newrelic/hooks/component_sklearn.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2010 New Relic, Inc.
```

Comment:
This file was renamed so I believe this was created as the result of a merge conflict. This file should be deleted.